test/gtest/ucs/test_async.cc

```diff
@@ -786,6 +852,14 @@ UCS_TEST_SKIP_COND_P(test_async_event_mt, multithread,
     EXPECT_GE(min_count, exp_min_count);
 }
 
+#if UCS_ENABLE_ASSERT
+UCS_TEST_SKIP_COND_P(test_async_event_mt, check_blocks_multithread,
+                     !(HAVE_DECL_F_SETOWN_EX))
```

Comment:
why need to check HAVE_DECL_F_SETOWN_EX?
also, is_blocked_test is not needed both in timer and event
I think it should be under test_async fixture

Reply:
> why need to check HAVE_DECL_F_SETOWN_EX?

removed

> also, is_blocked_test is not needed both in timer and event

yes, it can be tested only once - done

> I think it should be under test_async fixture

it requires swan/stop infrastructure implemented in `test_asyn_mt<T>`